modules/installation-osp-config-yaml.adoc

```diff
@@ -5,13 +5,12 @@
 [id="installation-osp-config-yaml_{context}"]
 = Sample customized `install-config.yaml` file for {rh-openstack}
```

Comment:
Existing content, but still suggest removing the backticks in the module heading here, per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata).